Quotient/events/roommessageevent.cpp

```diff
@@ -189,19 +198,51 @@ bool RoomMessageEvent::hasTextContent() const
                || msgtype() == MsgType::Notice);
 }
 
+std::unique_ptr<TextContent> RoomMessageEvent::textContent() const
```

Comment:
```suggestion
std::unique_ptr<TextContent> RoomMessageEvent::richTextContent() const
```
Because on plain-text-only events it will return a null pointer.